javaharness/java/arcs/android/AndroidArcsEnvironment.java

```diff
@@ -0,0 +1,32 @@
+package arcs.android;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import arcs.api.ArcsEnvironment;
+
+/**
```

Comment:
This class is now purely for controlling the lifecycle of the environment. However, it's actually not used for that. The AndroidHarnessController is now responsible for spinning up the web-shell/web view, but it doesn't implement ArcsEnvironment. So either we should get rid of ArcsEnvironment, or HarnessController should extend ArcsEnvironment?

Reply:
Good point. This is also my plan but I want to do it in a following PR so this PR is not too large.